__tests__/app.test.js

```diff
@@ -131,38 +131,113 @@ describe("GET /api/articles", () => {
 });
 
 describe("GET /api/articles/:article_id/comments", () => {
```

Comment:
One more test that this endpoint would really benefit from, is a valid but non existent article_id, e.g

GET '/api/articles/99999/comments'